README.md

```diff
@@ -1,5 +1,22 @@
-# Morpho Blue IRM
+# Morpho Blue IRMs
+
+Some interest rate models for Morpho Blue:
+- [AdaptiveCurveIRM](./src/AdaptiveCurveIrm.sol)
+
+## Resources
+
+- AdaptiveCurveIRM: [docs](https://www.notion.so/morpho-labs/Morpho-Blue-Documentation-Hub-External-00ff8194791045deb522821be46abbdc?pvs=4#d8269074bfd649009f28625a9caa38ea), [announcement article](https://morpho.mirror.xyz/aaUjIF85aIi5RT6-pLhVWBzuiCpOb4BV03OYNts2BHQ).
 
 ## Audits
 
-All audits are stored in the [audits](./audits/)' folder.
\ No newline at end of file
+All audits are stored in the [audits](./audits/)' folder.
+
+## Licences
+
+The primary licence is MIT, see [LICENCE](./LICENSE).
+
+## Getting started
+
+Install dependencies: `forge install`
+
```

Comment:
Don't even need this for running tests (and for what else would you need it ?)

Reply:
I think that you need it for forge-std